packages/e2e/package.json

```diff
@@ -6,9 +6,10 @@
   "scripts": {
     "clean": "rimraf node_modules",
     "dev": "TZ=UTC cypress open",
-    "test": "TZ=UTC cypress run --spec features/ui/components/authenticator/*.features",
+    "test:examples": "TZ=UTC cypress run --spec features/ui/hooks/*.features,features/ui/components/authenticator/*.features",
```

Comment:
Created this to run both hooks and authenticator tests, which both need the examples app to be running. I don't think it makes sense to have just a `test` command, since each test suite needs a different app running. Perhaps we should also make a test:docs command for the theme and future tests as well.